### ISSUE TITLE
Line height set for text editor not suitable for error display box

### ISSUE THREAD

**Issue by flukejones** [lapce/lapce#467]
The lines in the text area appear squashed if a line height that looks good for main editor is used.

This is illustrated by the screenshots in this issue: https://github.com/lapce/lapce/issues/466

The line height setting being used is 18, and font is `Comic Mono`, same result with other fonts like `Cascadia Code`.

**Comment by bugadani** [comment]
You're referring to the warnings list, right? Does it depend on line height at all?

**Comment by flukejones** [comment]
Yes it does. A large line height looks good there but terrible in the editor.

**Comment by bugadani** [comment]
It looks like spacing doesn't take multiple lines into account. I'll try and fix this, thanks for reporting.

**Comment by flukejones** [comment]
Here's some additional screenshots. I don't know if these issues don't exist on other platforms as I have only Linux installed on my PC's. But yeah, text seems to be a little wonky in a few areas.

![Screenshot from 2022-05-06 10-55-49](https://user-images.githubusercontent.com/2015870/167039731-8e1fdbf3-b8f7-4657-b2b3-a3146929828d.png)
![Screenshot from 2022-05-06 10-56-06](https://user-images.githubusercontent.com/2015870/167039739-4124d6fd-cdf7-4928-80f8-8237f7b15ddc.png)
![Screenshot from 2022-05-06 10-56-27](https://user-images.githubusercontent.com/2015870/167039743-8b7006da-2c3b-4831-b438-7ad4fbfc8897.png)
![Screenshot from 2022-05-06 10-56-55](https://user-images.githubusercontent.com/2015870/167039746-0174461e-3ca3-4a28-936c-e8ca0460358f.png)

**Comment by bugadani** [comment]
You're right, of course, but these other cases are different issues, with different causes. Don't worry, they will be fixed with time, but I don't think they are a priority right now.

**Comment by dzhou121** [comment]
This should be fixed now by the latest v0.1.0 release.

**Comment by bugadani** [comment]
Text does not overlap in the latest release but I think the right side of the error messages is still cut off so I'm going to keep this open a little while longer.
